Provider/IcsProvider.php

```diff
@@ -18,10 +18,6 @@ public function createTimezone(array $config = array(), $timezoneType = FALSE)
 
     public function createCalendar(Timezone $tz = null)
     {
-        if (is_null($tz)) {
```

Comment:
There is a problem with this modification because it introduces a BC Break.

Before this PR, the library created explicitly a Timezone when no timezone were given. With this modification we create a calendar without a timezone.

I don't know the good solution, perhaps a second parameter, boolean $allowNullTimezone=false or something like that. But this BC Break is a problem.

Reply:
Can you also add a phpdoc to explain why we introduces this possibility to have a null Timezone (because of Outlook)

Reply:
Can you also add a small comment in the readme of this bundle to explain that we introduce a second parameter in this method to manage the Outlook compatibility.